app/main.py

```diff
@@ -1 +1,47 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills: list[str] = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
```

Comment:
According to the checklist, you should use the `extend()` method or `+=` operator instead of `append()` to add multiple values to a list. This is not a bug, but a code style issue. Here is how you can do it:

python
self.skills += [skill]
```

or

```python
self.skills.extend([skill])
```

This way, if you decide to allow learning multiple skills at once in the future, you can easily modify your function to accept a list of skills.